### PR TITLE
fix cert manager timeout

### PR DESCRIPTION
**Pull request by zjaco13**
Adds a null resource to cert manager to stop the timeout when creating

### CODE REVIEW

**Comment by elamaran11** [comment]
@nimakaviani Please approve this. This is another bug we found while using reference-implementation.

File: terraform/cert-manager.tf [hunk 1]
```diff
@@ -2,17 +2,19 @@ resource "kubectl_manifest" "application_argocd_cert_manager" {
   yaml_body = templatefile("${path.module}/templates/argocd-apps/cert-manager.yaml", {
     REPO_URL = local.repo_url
   })
+}
 
+resource "null_resource" "wait_for_cert_manager" {
```
Comment (nabuskey) [inline terraform/cert-manager.tf:7]: null_resource is deprecated. let's use what's suggested in the doc here: https://registry.terraform.io/providers/hashicorp/null/latest/docs/resources/resource

**Review by nabuskey — CHANGES_REQUESTED** [review]
Thanks for this! Just a minor comment to make sure we aren't using something that's deprecated.

File: terraform/versions.tf [hunk 2]
```diff
@@ -18,5 +18,9 @@ terraform {
       source  = "alekc/kubectl"
       version = ">= 2.0.0"
     }
+    http = {
+      source = "hashicorp/http"
+      version = ">= 3.4.4"
+    }
```
Comment (nabuskey) [inline terraform/versions.tf:24]: Is this module needed?

**Review by nabuskey — COMMENTED** [review]
Thanks for making the change. Just have a question about the new module being added. Please also fix DCO so we can merge: https://github.com/apps/dco

**Review by nabuskey — APPROVED** [review]
LGTM. Thank you very much.

**Pull request merged**